src/PKSim.Core/Model/Template.cs

```diff
@@ -26,10 +26,10 @@ public enum TemplateType
       PopulationAnalysisField = 1 << 8,
       ObservedData = 1 << 9,
       PopulationSimulationAnalysisWorkflow = 1 << 10,
-      Observers = 1 << 11,
+      ObserverSet = 1 << 11,
```

Comment:
This enum is independent from the other one. Hence it was not renamed. I added a test